### ISSUE TITLE
Labor Categories

### ISSUE THREAD

**Issue by osi-vohra** [18F/its70-fs-epermit-scale-up#25]
# Question/Comment on the Forest Service RFP
Labor Categories

## Name and affiliation
Indu Vohra, Scrum Master, Optimal Solutions Inc.

## Section of RFP documents
5.1.1.1 Technical understanding and approach

## Question/Comment
Will the government allow use of schedule 70 labor categories to map with 18F categories for this procurement?

**Comment by hannahkane** [comment]
Yes. Vendors will be provided with a pricing table in which they should map their existing Schedule 70 labor categories to the roles that they are proposing for their team.
